scripts/check-code.sh

```diff
@@ -4,18 +4,18 @@
 set -eu
 set -x
 {
-    LINT_ARGS=(lambda_modules/*/* $(find raijin_scripts/ lambda_functions/ -iname '*.py'))
+    LINT_ARGS=(lambda_modules/*/* $(find raijin_scripts lambda_functions ! -path '*node_modules*' -name '*.py'))
 } &> /dev/null
 
 export PYTHONPATH=$PWD/lambda_modules/dea_es:$PWD/lambda_modules/dea_raijin${PYTHONPATH:+:${PYTHONPATH}}
 
 # Python linting
 pycodestyle "${LINT_ARGS[@]}"
 
-pylint -j 2 --reports no "${LINT_ARGS[@]}"
+#pylint -j 2 --reports no "${LINT_ARGS[@]}"
```

Comment:
Accident?

Reply:
No. It was being a complete pain and I'm going to fix it today. :)

Reply:
I wonder if it would be worth investigating flake8 again? Last I looked It seemed more pragmatic with its choice of checks. I like pylint's checks within an editor, but when used with CI it's quite a lot of maintenance (lots of noise, frequent changes). Or alternatively we pin the pylint version again.